atorch/examples/ds_3d_parallel/requirements.txt

```diff
@@ -0,0 +1 @@
+datasets>=2.14.6
```

Comment:
+ modelscope for dataset downloading.
\

Reply:
logger for install